File: learn-javascript-2/module_classes/classes.js
class Person {
  constructor(name) {
    this._name = name;
  }

  get name() {
    return this._name;
  }

  set name(value) {
    if (value.length > 2) {
      this._name = value;
    } else {
      console.log('Rejected');
    }
  }
}

var john = new Person('John');
console.log(john.name);
john.name = 'Le';
console.log(john.name);

File: 01-testing/dart/classes.dart
class Person {
  String name;

  Person(this.name);

  Person.create(this.name);

  void sayHello() {
    print("Hello.");
  }
}

class LediHildawan extends Person {
  LediHildawan(String name) : super(name)

  @override
  void sayHello() {
    print("Halo.");
  }
}

void main(List<String> args) {
  var ledihildawan = new LediHildawan("Ledi Hildawan");

  print(ledihildawan);
}
